orc8r/gateway/python/magma/magmad/config_manager.py

```diff
@@ -73,6 +74,34 @@ def get_request_args(self, stream_name: str) -> Any:
         )
         return mconfig_digest_proto
 
+    def _warn_if_versions_are_incompatible(self, agw_version, unpacked_mconfig):
+        # version should be in X.X.X format with only non-negative numbers allowed for X
+        VERSION_REGEX = re.compile(r"[0-9]+\.(?P<minor_version>[0-9]+)\.[0-9]+")
```

Comment:
It would be good to declare this as constant outside the function

Reply:
its not used outside of this function, so i wonder if its worth the code pollution?